docker/Dockerfile-dev

```diff
@@ -40,7 +38,7 @@ ADD collect_static.sh /
 ADD startup.sh /
 ADD test.sh /
 ADD manage /usr/local/bin/manage
-ADD manage /usr/local/bin/django-admin
+ADD django-admin /usr/local/bin/django-admin
```

Comment:
Now we have an actual manage.py script, so we can have wrappers for both tasks.  They are different in djangoland, where manage.py auto-sets your --settings value, essentially.  django-admin doesn't, so it can be used for more server-wide tasks if necessary.